src/core/MIGRATION.md

```diff
@@ -1198,7 +1198,7 @@ This table shows where these uiExports have moved to in the New Platform. In mos
 | `styleSheetPaths`            |                                                                                                                           |                                                                                                                                       |
 | `taskDefinitions`            |                                                                                                                           | Should be an API on the taskManager plugin.                                                                                           |
 | `uiCapabilities`             | [`core.application.register`](/docs/development/core/public/kibana-plugin-public.applicationsetup.register.md)            |                                                                                                                                       |
-| `uiSettingDefaults`          |                                                                                                                           | Most likely part of server-side UiSettingsService.                                                                                    |
+| `uiSettingDefaults`          | [`core.uiSettings.register`](docs/development/core/server/kibana-plugin-server.uisettingsservicesetup.md) |                                                                    |
```

Comment:
Needs leading slash to make link point to root of repo rather than relative path from `src/core/`

```suggestion
| `uiSettingDefaults`          | [`core.uiSettings.register`](/docs/development/core/server/kibana-plugin-server.uisettingsservicesetup.md) |                                                                    |
```